plugins/vpp/l3plugin/arp_config_test.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package l3plugin_test
+
+import (
+	"git.fd.io/govpp.git/adapter/mock"
+	"git.fd.io/govpp.git/core"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/ligato/vpp-agent/idxvpp/nametoidx"
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/ip"
+	"github.com/ligato/vpp-agent/plugins/vpp/ifplugin/ifaceidx"
+	"github.com/ligato/vpp-agent/plugins/vpp/l3plugin"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/l3"
+	"github.com/ligato/vpp-agent/tests/vppcallmock"
+	. "github.com/onsi/gomega"
+	"testing"
+)
+
+// Test ArpConfigurator initialization
+func TestArpConfiguratorInit(t *testing.T) {
+	RegisterTestingT(t)
+	ctx := &vppcallmock.TestCtx{
+		MockVpp: &mock.VppAdapter{},
+	}
+	connection, _ := core.Connect(ctx.MockVpp)
+	defer connection.Disconnect()
+
+	plugin := &l3plugin.ArpConfigurator{}
+
+	// Test init
+	err := plugin.Init(logging.ForPlugin("test-log", logrus.NewLogRegistry()), connection, nil, false)
+	Expect(err).To(BeNil())
+	err = plugin.Close()
+	Expect(err).To(BeNil())
+}
+
+var arpTableEntries = []l3.ArpTable_ArpEntry{
```

Comment:
Try to avoid globals for test. Either create some helper function which will take arguments (interface, ip, mac..) and create object for you or define these explicitly for each tests.